src/lib.rs

```diff
@@ -72,6 +81,21 @@ pub fn format_cargo_crates(packages: Vec<Package>, mode: AlignmentMode) -> Strin
                     name_width = 28,
                     version_width = 8
                 ),
+                AlignmentMode::Justify => {
+                    let version_len = package.version.to_string().len();
+                    let space_width =
+                        package_max_width - package.name.as_str().len() - version_len + 5;
```

Comment:
Magic number `5` should be explained in comments.

Reply:
The magic number is what I required to make it match against one of the outputs when testing. This should *probably* be configurable, but it's a good number and avoids parsing and testing something unnecessarily.